### ISSUE TITLE
Readme

### ISSUE THREAD

**Issue by Normal-repo** [Normal-repo/CIFAR-10-Classification#1]
# CIFAR-10 Image Classification

This repository contains image classification models for the CIFAR-10 dataset using **Custom CNN** and **ResNet50 (pretrained on ImageNet)**.

---
## 🔹 Project Overview

This repository implements **image classification on the CIFAR-10 dataset** using:

- **Custom CNN** (with BatchNorm and Dropout)
- **ResNet50 pretrained on ImageNet** (fine-tuned for CIFAR-10)

**Goal:** Classify 32x32 images into 10 classes: airplane, automobile, bird, cat, deer, dog, frog, horse, ship, truck.

---

## 🧩 Project Structure

 CIFAR10_CNN_ResNet
├─ CIFAR10_CNN.ipynb # Training & evaluation of custom CNN
├─CIFAR10_ResNet.ipynb # Training & evaluation of ResNet50
├─ cnn_model.h5 # Trained custom CNN model
├─ resnet_model.h5 # Trained ResNet50 model
├─README.md

---

## 📊 Model Performance

| Model           | Validation Accuracy |
|-----------------|------------------|
| Custom CNN      | ~82–83%          |
| ResNet50        | ~88–90%          |

---

## 🖼️ Example Predictions

| Input Image | Predicted Class |
|------------|----------------|
| ![airplane](examples/airplane.png) | Airplane |
| ![cat](examples/cat.png) | Cat |
| ![truck](examples/truck.png) | Truck |

---
